core/appmodule/v2/handlers.go

```diff
@@ -44,7 +44,11 @@ type PostMsgRouter interface {
 	// the provided name.
 	Register(msgName string, handler PostMsgHandler)
 	// RegisterGlobal will register a global message handler hooking after the execution of any message.
-	RegisterGlobal(handler PreMsgHandler)
+	RegisterGlobal(handler PostMsgHandler)
+}
+
+type HasPostMsgHandlers interface {
```

Comment:
What is this? Where is it used?

Reply:
oh got mixed up with another PR, i'll remove this

Reply:
Let's wait until @testinginprod re-implement this: https://github.com/cosmos/cosmos-sdk/pull/21239#discussion_r1713446086 and not add anything in core in this PR.

Reply:
removed